src/pages/404.tsx

```diff
@@ -6,19 +6,121 @@
  */
 
 import React from 'react';
-
+import { css } from 'styled-components';
 import SEO from 'components/SEO';
+import { useStaticQuery, graphql } from 'gatsby';
+import Img from 'gatsby-image';
 import RootLayout from 'layouts/Root';
 import MaxWidthLayout from 'layouts/MaxWidth';
+import { getColor } from '@zendeskgarden/react-theming';
+import { Grid, Row, Col } from '@zendeskgarden/react-grid';
+import { XL, LG } from '@zendeskgarden/react-typography';
+import { StyledH1 } from 'components/MarkdownProvider/components/Typography';
+
+const NotFoundPage: React.FC = () => {
+  const { notFoundImage } = useStaticQuery(
+    graphql`
+      query {
+        notFoundImage: abstractAsset(layerName: { eq: "content-voice-tone-map" }) {
```

Comment:
`{ eq: "general-error-404" }` Should do the trick

Reply:
brilliant.